tools/src/bin/output-file.rs

```diff
@@ -335,22 +335,58 @@ fn main() {
                     accel_key: None,
                     copyable: false,
                 });
-                if let Some(ref hg_root) = tree_config.paths.hg_root {
+
+                let gh_log_link = tree_config
+                    .paths
+                    .github_repo
+                    .as_ref()
+                    .map(|gh_root| format!("{}/commits/HEAD/{}", gh_root, encoded_path));
+                let has_gh_log_link = gh_log_link.is_some();
+                if let Some(link) = gh_log_link {
                     vcs_panel_items.push(PanelItem {
-                        title: "Log".to_owned(),
-                        link: format!("{}/log/tip/{}", hg_root, encoded_path),
+                        title: "Git log".to_owned(),
+                        link,
                         update_link_lineno: "",
                         accel_key: Some('L'),
                         copyable: true,
                     });
+                }
+
+                let hg_log_link = tree_config
+                    .paths
+                    .hg_root
+                    .as_ref()
+                    .map(|hg_root| format!("{}/log/tip/{}", hg_root, encoded_path));
+                if let Some(link) = hg_log_link {
+                    vcs_panel_items.push(PanelItem {
+                        title: "Mercurial log".to_owned(),
+                        link,
+                        update_link_lineno: "",
+                        accel_key: (!has_gh_log_link).then_some('L'),
```

Comment:
So unfortunately I think we need to make the logic a little weirder than this.  As discussed on chat.mozilla.org, I think if we have both "Git log" and "Mercurial log" we want the hotkey to go to Mercurial, even though we want to visually display "Git log" first to try and bias people who don't have a preference to use Git log[1].

1: Specifically:
- We expect git log to be faster based on hg.mozilla.org's load problems right now.
- I'm hearing that we want to discourage people from using hg.mozilla.org as much as possible
- Because we are moving from hg to git for m-c, we do expect "Git log" to become the only non-searchfox option available in the medium term.  (That said, as I noted in the channel, ideally I'll be adding a new searchfox-based option that maybe could end up being our new default.)

Reply:
The latest version should now give the priority to the Mercurial log for the L hotkey.

I've also removed the hardcoding of hotkeys to nodes in panel.js. In the first version of this PR, I assumed changing `PanelItem::accel` was enough and broke the hotkey. Now changing `PanelItem::accel` should actually be enough.